anthos-bm-gcp-terraform/test/unit/main_test.go

```diff
@@ -463,6 +468,25 @@ func TestUnit_MainScript_ValidateDefaults(goTester *testing.T) {
 		terraformPlan.Variables.InstanceCount.Value["worker"],
 		"Variable does not match expected default value: instance_count.worker.",
 	)
+
+	defaultGpuSetup := util.Gpu{
+		Count: 0,
+		Type:  "",
+	}
+	// verify input variable gpu in plan matches the default value
+	assert.Equal(
+		goTester,
+		defaultGpuSetup.Count,
+		terraformPlan.Variables.Gpu.Value.Count,
+		"Variable does not match expected default value: abm_cluster_id.",
+	)
+
+	assert.Equal(
+		goTester,
+		defaultGpuSetup.Type,
+		terraformPlan.Variables.Gpu.Value.Type,
+		"Variable does not match expected default value: abm_cluster_id.",
```

Comment:
Should this be: "Variable does not match expected default value: gpu.type" ?

Reply:
Yes, Indeed. Fixed in https://github.com/GoogleCloudPlatform/anthos-samples/pull/68/commits/a1db93c183510ef8737d9bed89a929722c76470c